protocol/messenger_backup_handler.go

```diff
@@ -205,12 +205,22 @@ func (m *Messenger) handleBackedUpSettings(message *protobuf.SyncSetting) error
 		return nil
 	}
 
-	if settingField != nil && m.config.messengerSignalsHandler != nil {
-		response := wakusync.WakuBackedUpDataResponse{
-			Setting: settingField,
+	if settingField != nil {
+		if message.GetType() == protobuf.SyncSetting_PREFERRED_NAME && message.GetValueString() != "" {
```

Comment:
@qfrank the changes look good, I have only 2 general questions:
- Do we need to display the ens name at all on the login screen or we display the display name and use the preferred ens name after login? (This is more about whether is that a request or not?)
- What in case the user changes the setting or removes the ens name but the backup still refers to the preferred ens name which is not in use anymore?

Reply:
> Do we need to display the ens name at all on the login screen or we display the display name and use the preferred ens name after login? (This is more about whether is that a request or not?)

I had the same feeling, and @cammellos made a clarification [here](https://github.com/status-im/status-mobile/pull/17127#issuecomment-1717879690)

> What in case the user changes the setting or removes the ens name but the backup still refers to the preferred ens name which is not in use anymore?

No process in this case , mobile doesn't support remove ENS ATM @saledjenic